layouts/partials/json-schema/resolve-refs.html

```diff
@@ -1,7 +1,10 @@
 {{/*
```

Comment:
Should we do like in #1749 and allow to override properties named `description` and `summary`?

Reply:
No, I don't think that's necessary. The validator will complain if anyone tries to override other properties, I think?

Reply:
I doesn't, afaict.

Reply:
oh. bother. Well, maybe something to think about adding into our validation scripts in future.